grease-cli/tests/sanity/pass/startup-override/test.c

```diff
@@ -14,6 +14,5 @@ void test(uint8_t *buf, size_t sz) {
   memset(buf, 'a', sz);
 }
 // arm: ok()
-// TODO: Why does PPC fail here?
-// ppc32: check "PPCInvalidInstruction"
+// ppc32: check "Could not identify block at"
```

Comment:
This is happening because of [limitations in `macaw`'s heuristics for finding PLT stub addresses](https://github.com/GaloisInc/macaw/issues/386). To work around these limitations, we need to manually tell `grease` where the `memset` PLT stub is on PowerPC. To do so: add the following to `// ppc32: flags ...` above:

```c
// ppc32: flags {..., "--plt-stub", "0x10000220:memset"}
```